server.js

```diff
@@ -1210,6 +1210,37 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Package Control integration.
+camp.route(/^\/packagecontrol\/(.*)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Could you add a prefix to distinguish between different packagecontrol badges? Maybe `/packagecontrol/dt` for total downloads (we might have `/packagecontrol/dd` for daily downloads, etc.)?